src/main/java/org/interledger/InterledgerPacket.java

```diff
@@ -0,0 +1,245 @@
+package org.interledger;
```

Comment:
No objections there - I'm going to add a "usage" page on the wikie (for now) and once we get #33 worked out, we can disable the wiki.